### PR TITLE
New: Persist search settings in add new series

### PR DESCRIPTION
**Pull request by Snarkenfaugister**
#### Database Migration
NO

#### Description
Persist `searchForCutoffUnmetEpisodes` and `searchForMissingEpisodes` setting values in local cache.

#### Issues Fixed or Closed by this PR

* #4245

### CODE REVIEW

**Comment by bakerboy448** [comment]
For your commits if you do 

“Fixed: Blah Blah Blah
<new line>
Closes #GHI”

once merged itll 1) show as fixed on the change log and 2) reference the GHI on the change log

Appreciate the help though!

(force push w/ lease if you reword the commits)

File: frontend/src/Store/Actions/addSeriesActions.js [hunk 1]
```diff
@@ -37,6 +37,8 @@ export const defaultState = {
     languageProfileId: 0,
     seriesType: seriesTypes.STANDARD,
     seasonFolder: true,
+    searchForMissingEpisodes: true,
```
Comment (markus101) [inline frontend/src/Store/Actions/addSeriesActions.js:40]: These should both default to false like they do currently.

**Review by markus101 — CHANGES_REQUESTED** [review]
Just the default comment and then good to land.

**Comment by markus101** [comment]
Thanks!

**Pull request merged**